src/averageAges.js

```diff
@@ -1,66 +1,89 @@
 'use strict';
 
+const MALE_SEX = 'm';
+const FEMALE_SEX = 'f';
+const CENTURY_DIVIDER = 100;
 /**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
  * @param {object[]} people
  * @param {number} century - optional
  *
  * @return {number}
  */
+/* eslint-disable no-console */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
-}
+  const mens = people.filter(
+    person =>
+      isMale(person)
+        && (century
+          ? (Math.ceil(person.died / CENTURY_DIVIDER) === century)
+          : true)
+  );
+
+  return +getAverageAge(mens).toFixed(2);
+};
 
 /**
- * Implement calculateWomenAverageAge function
- *
- * Function returns average age of women in array. If `withChildren` is
- * specified then function calculates average age only for women with children
- *
- * Hint: To check if a woman has children you should find someone who mention
- * her as mother.
- *
  * @param {object[]} people
  * @param {boolean} withChildren - optional
  *
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const womens = people.filter(
```

Comment:
women - жінки, womens - жіночий
```suggestion
  const women = people.filter(
```